rasa/nlu/classifiers/logistic_regression_classifier.py

```diff
@@ -0,0 +1,190 @@
+import logging
+from typing import Any, Text, Dict, List, Type
+
+from joblib import dump, load
```

Comment:
Since `dump` and `load` could be occurring in other modules as well and lead to shadowing, it might be better to call them by their full module path, as per [here](https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#d5fb5c6142394ff99505dc8160255ef5):

```
import joblib
joblib.dump(...)
```